build-tools/packages/build-cli/src/data/exclusions.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Is this a copy of the existing file, or a move? Either way, I say leave the existing file in place, and then when we update the build scripts in package.json to use the new CLI we can move the file then.